rest-client-base/pom.xml

```
@@ -49,7 +47,7 @@
         <dependency>
             <groupId>io.getlime.core</groupId>
             <artifactId>rest-model-base</artifactId>
-            <version>1.5.2</version>
+            <version>1.6.0-SNAPSHOT</version>
```

Just one little thing, here we may use 
```suggestion
            <version>${project.version}</version>
```